go.mod

```diff
@@ -3,61 +3,88 @@ module github.com/prometheus/prometheus
 go 1.13
 
 require (
-	contrib.go.opencensus.io/exporter/ocagent v0.6.0 // indirect
-	github.com/Azure/azure-sdk-for-go v23.2.0+incompatible
-	github.com/Azure/go-autorest v11.2.8+incompatible
-	github.com/alecthomas/units v0.0.0-20190717042225-c3de453c63f4
-	github.com/aws/aws-sdk-go v1.25.48
+	cloud.google.com/go v0.53.0 // indirect
+	github.com/Azure/azure-sdk-for-go v39.1.0+incompatible
+	github.com/Azure/go-autorest/autorest v0.9.5
+	github.com/Azure/go-autorest/autorest/adal v0.8.2
+	github.com/Azure/go-autorest/autorest/to v0.3.0 // indirect
+	github.com/Azure/go-autorest/autorest/validation v0.2.0 // indirect
+	github.com/alecthomas/units v0.0.0-20190924025748-f65c72e2690d
+	github.com/armon/go-metrics v0.3.2 // indirect
+	github.com/asaskevich/govalidator v0.0.0-20200108200545-475eaeb16496 // indirect
+	github.com/aws/aws-sdk-go v1.29.4
 	github.com/cespare/xxhash v1.1.0
 	github.com/dgryski/go-sip13 v0.0.0-20190329191031-25c5027a8c7b
 	github.com/edsrzf/mmap-go v1.0.0
-	github.com/go-kit/kit v0.9.0
-	github.com/go-logfmt/logfmt v0.4.0
-	github.com/go-openapi/strfmt v0.19.2
-	github.com/gogo/protobuf v1.2.2-0.20190730201129-28a6bbf47e48
+	github.com/go-kit/kit v0.10.0
```

Comment:
Fixes #6795

Reply:
I've edited the PR description to reference this issue